tools/test/run_ci_server.sh

```diff
@@ -13,7 +13,7 @@ function main() {
   surpress_on_success docker-compose -f docker-compose-test.yml up --force-recreate -d weaviate \
     janus index db genesis_fake remote_weaviate_fake remote_weaviate_fakes_with_test_schema genesis
 
-  MAX_WAIT_SECONDS=60
+  MAX_WAIT_SECONDS=720
```

Comment:
Interesting attempt to fix this problem ;)

As discussed on Slack, I'll update the settings, so that we don't download the c11y in test/CI settings, but rather use the minimal dev version from the file.

Reply:
haha, true, I'll change it back